test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpErrorTest/WithHelperDirectiveProducesError.stree.txt

```diff
@@ -1,5 +1,8 @@
-Expression block - Gen<Expr> - 7 - (0:0,0)
-    Transition span - Gen<None> - [@] - SpanEditHandler;Accepts:None - (0:0,0) - Tokens:1
-        SyntaxKind.Transition;[@];
-    Code span - Gen<Expr> - [helper] - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[RTD];K14 - (1:0,1) - Tokens:1
-        SyntaxKind.Identifier;[helper];
+CSharpCodeBlock - [0..7)::7 - [@helper]
```

Comment:
curious, what error do we provide?

Reply:
(1,2): Error RZ1002: The helper directive is not supported.